apps/emqx_management/src/emqx_management.app.src

```diff
@@ -2,7 +2,7 @@
 {application, emqx_management, [
     {description, "EMQX Management API and CLI"},
     % strict semver, bump manually!
-    {vsn, "5.0.10"},
+    {vsn, "5.0.11"},
```

Comment:
I'm a bit unsure about the versioning here, but I assume that the semver part is reflecting _internal_ changes, and not HTTP API changes/breakage. Thus I simply bump the patch number.

Reply:
There's no need to bump those versions, unless the CI check which runs `scripts/apps-version-check.sh` complains about version bumping.
Those are mostly for appup/relup tracking.

Reply:
Yep, In this case the CI complained so I bumped :) Thanks for the info.